### PR TITLE
Nested arrays

### PR DESCRIPTION
**Pull request by amosr**
! @tranma @jystic 
/jury approved @tranma @jystic

### CODE REVIEW

File: data/sea/30-array.h [hunk 1]
```diff
@@ -315,3 +315,10 @@ MAKE_ARRAY(ARRAY(itime))
 MAKE_ARRAY(ARRAY(istring))
 MAKE_ARRAY(ARRAY(iunit))
 #endif
+
+
+typedef void* iany_t;
```
Comment (tranma) [inline data/sea/30-array.h]: an `Object`! It's top, it's everything except the things that matters (chars, ints) :D

File: data/sea/54-zebra.h [hunk 2]
```diff
@@ -138,66 +151,206 @@ static ierror_msg_t zebra_translate
...
+                for (int offset_ix = 0; offset_ix != offset; ++offset_ix) {
+                    ARRAY_PAYLOAD(iany, dst[offset_ix])[nested_ix] = tmp_into[offset_ix];
+                }
+            }
+
+            return offset;
+        }
     }
+    fprintf (stderr, "Fatal error: unknown Zebra table tag: %u\n", src->tag);
+    exit (1);
```
Comment (tranma) [inline data/sea/54-zebra.h]: this function looks good

**Comment by tranma** [comment]
:eye:'d the C
👍

**Comment by jystic** [comment]
👍 seems to working in production

**Pull request merged**